debezium-connector-sqlserver/src/test/java/io/debezium/connector/sqlserver/util/TestHelper.java

```diff
@@ -210,16 +210,18 @@ private static void dropTestDatabase(SqlServerConnection connection) throws SQLE
     }
 
     public static SqlServerConnection adminConnection() throws SQLException {
-        SqlServerConnection connection = new SqlServerConnection(TestHelper.adminJdbcConfig(), Clock.system(), SourceTimestampMode.getDefaultMode(),
+        JdbcConfiguration jdbcConfig = TestHelper.adminJdbcConfig();
+        SqlServerConnection connection = new SqlServerConnection(jdbcConfig, Clock.system(), SourceTimestampMode.getDefaultMode(),
                 new SqlServerValueConverters(JdbcValueConverters.DecimalMode.PRECISE, TemporalPrecisionMode.ADAPTIVE, null));
-        connection.execute("USE " + connection.config().getDatabase());
+        connection.execute("USE " + jdbcConfig.getString(SqlServerConnectorConfig.DATABASES_FIELD_NAME));
```

Comment:
How will it work if the list is plural? Do we really need the `USE` here given that the connector works with multiple databases and should be stateless?

Reply:
This test helper works with just 1 specific database (testdb). Not specifying it here will require require specifying it in every direct sql statement. And we have lots of them.

Reply:
> This test helper works with just 1 specific database (testdb)

Should we wrap this into a "guard" method? It could be even put into `SqlServerConnectorConfig` which would allow making the newly introduced constant private as it's supposed to be. If the configuration contains more than one database, this method should fail right away.